docs/source/tutorials.rst

```diff
@@ -40,12 +40,19 @@ To run the tutorials in the ExaWorks SDK Docker container:
    containers you started to execute the RADICAL-Cybertools tutorial.
 
 
-SDK Core Components Tutorials
------------------------------
-
-* `Flux <tutorials/flux.ipynb>`_
-* `Parsl <tutorials/parsl/parsl.ipynb>`_
-* `PSI/J <tutorials/psij.ipynb>`_
-* `RADICAL-Cybertools <tutorials/rct.ipynb>`_
-* `Swift/T <tutorials/swift-t/swift-t.ipynb>`_
+
+SDK Tutorials
+-------------
+
+.. * `Flux <tutorials/flux.ipynb>`_
+.. * `Parsl <tutorials/parsl.ipynb>`_
+.. * `PSI/J <tutorials/psij.ipynb>`_
+.. * `RADICAL-Cybertools <tutorials/rct.ipynb>`_
+.. * `Swift/T <tutorials/swift.ipynb>`_
```

Comment:
Regarding subdirectories for Parsl and Swift/T:
- Parsl: In a corresponding docker container, we do load an additional tutorial repo, thus it is good to have a dedicated directory, where to keep it (here is a [link](https://github.com/ExaWorks/SDK/blob/8e112326b841705cf5ba60cbc18152de8e5ccd8b/docker/tutorials/Dockerfile#L46-L47) from PR #162 )
- Swift/T: Justin's comment regarding this - https://github.com/ExaWorks/SDK/pull/161#issuecomment-1409576597

Reply:
just for records as what we agreed on: keep documentation this way and use sub-directories for tutorials-container if needed